WalletWasabi.Fluent/ViewModels/Dialog/IDialogHost.cs

```diff
@@ -0,0 +1,26 @@
+#nullable enable
```

Comment:
Why not enable it on the project level like this: https://github.com/zkSNACKs/WalletWasabi/blob/78da80faee36f5d1f940d5975349f7dc4c844c97/WalletWasabi.Gui/WalletWasabi.Gui.csproj#L8
I guess that is what @danwalmsley  meant with  https://github.com/zkSNACKs/WalletWasabi/pull/4436#discussion_r496489371